Azure Cloud Adoption Framework with Datadog Guide (#13044)
DataDog/documentation: content/en/integrations/guide/azure-cloud-adoption-framework.md

For organizations that are following Azure's Cloud Adoption Framework through their migration journey, this document will guide you and your teams to...

1. Prepare your Datadog account for your teams to efficiently execute their workload migrations (a "plan" stage focus)
2. Use Datadog to measure the health of your original environments and new workloads as your teams launch them in your new Landing Zones (a "migration" stage focus). 

# Plan

During the plan stage of your migration, you will want to prepare your Datadog account to effectively monitor your new workloads as soon as they are migrated to your Azure account. What you will do here...

1. Enable Datadog-Azure integrations that your new workloads will use to indicate their performance and health
2. Document a tagging strategy that your teams will use to describe their workloads as they migrate
3. Configure dashboards that stakeholders can use to follow migration progress and understand the overall health of the new workloads
4. Establish communication channels for incident response

## Enable Datadog-Azure Integrations

If you don't yet have a Datadog account, you can [start a free 2-week trial from this link][1].

> Datadog and Azure partner together to offer Datadog's services from within your Azure account. For each of your landing zones, you can create a Datadog Resource to link your Datadog account to your Azure account and access your observability data from within Azure. [This guide][2]will walk you through that process, and will help you determine which of your Azure data to collect into Datadog.

Author: thanks for asking -- the link for that guide should be this `/integrations/azure/?tab=link&site=us3`

In some cases it may make sense to visualize the performance of your legacy environment and new workloads side-by-side. You can find an example of such a dashboard by taking the following steps:

1. In your Datadog account, [create a new dashboard][22].
2. Select the 'cog' at the top right corner
3. Select 'import dashboard content' ([setting documented here][23])
4. Paste in the JSON definition of the dashboard found [here][24]

## Send Actionable Alerts to Workload Owners

As you migrate your workloads, you will want to make sure the right people can be automatically alerted when important performance KPIs thresholds are crossed. To do this, create monitors in Datadog that constantly observe the health of your workloads and trigger notifications to your communication channels, wether over MS Teams, Slack, paging services, or ticketing systems.

> We recommend reviewing [this guide on how to set up actionable and effective alerts][25]. If your tags map effectively to dedicated investigative channels (e.g, if you have a Teams channel for every "owner" or "team" or "workspace", then you can configure your monitors to [dynamically notify the appropriate communication channel][10] with monitor template variables. 

Reviewer: Is this guide link to /monitors?

```suggestion
We recommend reviewing [this guide on how to set up actionable and effective alerts][25]. If your tags map effectively to dedicated investigative channels (e.g, if you have a Teams channel for every "owner" or "team" or "workspace", then you can configure your monitors to [dynamically notify the appropriate communication channel][10] with monitor template variables. 
```

Author: Yep, just `/monitors/`

---
title: Azure Cloud Adoption Framework with Datadog
kind: guide
further_reading:
  - link: "/integrations/azure/"
    tag: "Documentation"
    text: "Datadog-Azure Integration"
---

## Overview

Migrating your services from on-premises or other cloud environments to a new cloud environment can be risky. 

Azure's Cloud Adoption Framework has a proven strategy for successful migrations and Datadog's monitoring ensures safe, speedy migrations by measuring the health of your new and original environments in the same system of observability and incident response. 

You can:

1. Prepare your Datadog account for your teams to efficiently execute their workload migrations. This is the "planning" stage.
2. Use Datadog to measure the health of your original environments and new workloads as your teams launch them in your new landing zones. This is the "migrating" stage. 

> This guide documents the migration process for organizations that follow Azure's Cloud Adoption Framework.

Reviewer: ```suggestion
This guide documents the migration process for organizations that follow Azure's Cloud Adoption Framework.

If you don't have a Datadog account yet, you can [start a two-week trial][2].
```

## Planning

When planning your migration, prepare your Datadog account to monitor your new workloads as soon as they are migrated into your Azure account by doing the following:

1. Enable the [Datadog-Azure integration][1] so your new workloads can indicate their performance and health.
2. Document a tagging strategy that your teams can use to describe their workloads as they migrate.
3. Configure dashboards that stakeholders can use to follow along the migration progress and understand the overall health of the new workloads.
4. Establish communication channels for incident response.

### Enable Datadog-Azure Integrations

Datadog and Azure have partnered together to offer Datadog's services within your Azure account. For each one of your landing zones, you can create a Datadog resource to link your Datadog account with your Azure account and access your observability data in Azure. 

> If you don't have a Datadog account yet, [start a two-week trial][2]. For more information about this process, see the [Datadog-Azure Integration][1].

Reviewer: ```suggestion
For more information about this process to help you determine what Azure data you want to collect in Datadog, see the [Microsoft Azure documentation][1].
```
